services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/client/grpc/DataPlaneClientImpl.java

```diff
@@ -19,44 +19,97 @@
 import com.futurewei.alcor.dataplane.config.grpc.GoalStateProvisionerClient;
 import com.futurewei.alcor.dataplane.entity.HostGoalState;
 import com.futurewei.alcor.schema.Goalstate;
+import com.futurewei.alcor.schema.Goalstateprovisioner;
+import io.netty.util.concurrent.DefaultThreadFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
+import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.concurrent.*;
+import java.util.stream.Collectors;
 
 @Component
 public class DataPlaneClientImpl implements DataPlaneClient {
 
     private int grpcPort;
 
+    private static ExecutorService executor = new ThreadPoolExecutor(100, 200, 50, TimeUnit.SECONDS,
```

Comment:
As discussed in the open-source meeting, let us use Alcor Thread pool if possible. Please refer to PR #424 for details.

Reply:
It's only test code. And the original thread pool provides more flexibility.

Reply:
Well. Which kinds of features are available in original thread pool but not in Alcor thread pool? 

Meanwhile, we plan to make this more than test code :-)